testing/scripts/seldon_e2e_utils.py

```diff
@@ -348,26 +352,32 @@ def rest_request_ambassador(
     if names is not None:
         payload["data"]["names"] = names
 
-    if namespace is None:
+    if method == "predict":
         response = requests.post(
             "http://"
             + endpoint
             + "/seldon/"
+            + namespace
```

Comment:
Can't `namespace` be `None` here? TBH I don't think we ever ignore it, but we could just set a default on the args.

Reply:
You are right, in theory it shouldn't because in the case of e2e tests it would always be at least "default", so I'll add that as default.

Completely disjoint to this, but still worth mentioning, in the operator we actually allow for the concept of "global" ambassador routes, which basically don't have the namespace as part of the path, however I have another PR that will remove this as Istio doesn't really have this functionality and it would be good to fully align the paths and terminology across both, so if this change goes ahead there shouldn't be a case where namespace can be none.